cli/azd/cmd/account.go

```diff
@@ -0,0 +1,246 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/azure/azure-dev/cli/azd/internal"
+	"github.com/azure/azure-dev/cli/azd/pkg/account"
+	"github.com/azure/azure-dev/cli/azd/pkg/commands"
+	"github.com/azure/azure-dev/cli/azd/pkg/environment/azdcontext"
+	"github.com/azure/azure-dev/cli/azd/pkg/input"
+	"github.com/azure/azure-dev/cli/azd/pkg/output"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func NewAccountCmd(rootOptions *internal.GlobalCommandOptions) *cobra.Command {
```

Comment:
Would it be possible to cleanly remove the `cmd` and stash it in `git` until we need to introduce it? I'm a little worried about keeping "unsurfaced" code around as it introduces maintenance burden without actually being used in the product... From our chat, it's unlikely that we'd follow `az`'s account structure and would end up rewriting when introducing it too.

Reply:
done.